### ISSUE TITLE
What do you think?

### ISSUE THREAD

**Issue by sudodaksh** [kashifmin/deeplearning_challenge#1]
I checked your model, I think the data provided by them is not quality and very less in amount so it doesn't make any sense to build a model over it and I'll be better of using a pre-trained one.
But still, It's getting 70% accuracy on training data and probably 25-26% over training data.

I built a Pretty large model (I built a few models to be exact) on this but the maximum acc I was able to recieve over test data was 36%

I think the people ranking on top are probably using a Pre-trained model of some sort?

What do you think?

**Comment by kashifmin** [comment]
Actually there is silly list indexing error in this model. I fixed and some other changes, I'm able to get 98% accuracy on training data. But ofcourse it's overfitting.

**Comment by sudodaksh** [comment]
Yes same with me, I got 99.7% max accuracy over training data but testing sucks, I was getting 36-37% ish acc

**Comment by junaid1460** [comment]
Prolly we should collect some more data for those labels(Google image search?).

**Comment by kashifmin** [comment]
Maybe. Pretrained models are what those top scorers seem to be using :/
